LabApiUtilities/build.gradle

```diff
@@ -75,9 +75,14 @@ dependencies {
 
     implementation "com.nimbusds:nimbus-jose-jwt:$rootProject.ext.nimbusVersion"
     implementation "com.microsoft.azure:msal4j:$rootProject.ext.msal4jVersion"
+    implementation 'com.microsoft.azure:adal4j:1.6.7'
```

Comment:
Shahzaib, do we want to add this in version in versions.gradle file and reference it here?

Reply:
Yes, but I didn't do it on purpose because this requires creating 4 additional PRs (one in ADAL, MSAL, Broker, and Android-Complete) and is a tedious task. I think we need to solve this problem in a better way. I have some ideas and therefore I'll leave the version hard coded here now and try to resolve this task in a different way.